spec/support/seeds/seeds_e2e.rb

```diff
@@ -112,6 +112,34 @@
 
 ##############################################################################
 
+num_search_test_users = 7
+
+Array(1..num_search_test_users).each do |i|
+  seeder.create_if_doesnt_exist(User, "email", "user#{i}@forem.com") do
+    user = User.create!(
+      name: "Search user #{i}",
+      email: "user#{i}@forem.com",
+      username: "search_user_#{i}",
+      summary: Faker::Lorem.paragraph_by_chars(number: 199, supplemental: false),
+      profile_image: File.open(Rails.root.join("app/assets/images/#{rand(1..40)}.png")),
+      website_url: Faker::Internet.url,
+      email_comment_notifications: false,
+      email_follower_notifications: false,
+      confirmed_at: Time.current,
+      password: "password",
+      password_confirmation: "password",
+      saw_onboarding: true,
+      checked_code_of_conduct: true,
+      checked_terms_and_conditions: true,
+      editor_version: "v2",
+    )
+
+    user.add_role(:pro)
```

Comment:
Might need some advice on this one @nickytonline - I'm finding that if I don't add a role, the users created in this file aren't showing up in the `/search` call. Rather than adding a role maybe there's a step with elasticsearch I'm missing?

Reply:
This is odd, `:pro` is a role related to the analytics page 🕵️

Reply:
I just chose 'pro' as a random role 😄  - adding any role here seems to allow the user to show up in the e2e test search. But tbh I don't think it should be needed at all - users without roles show up fine in the non-e2e environment 🙃

Reply:
@aitchiss Random guess: the e2e environment does not automatically index to ES but adding a role triggers a re-index. Something in the `rails_helper` or `spec_helper` takes care of this for the "normal" test environment.

Reply:
that would make sense @citizen428 - thanks!

Reply:
seems to be the same issue causing the Listings E2E test to now fail (empty search results returned for the listings page instead of the listing in the seeds file)